src/reuse/modules/nonUi5/element.ts

```diff
@@ -1,11 +1,13 @@
 import { Element } from "../../../../@types/wdio";
 import { VerboseLoggerFactory } from "../../helper/verboseLogger";
+import ErrorHandler from "../../helper/errorHandler";
 /**
  * @class element
  * @memberof nonUi5
  */
 export class ElementModule {
   private vlf = new VerboseLoggerFactory("nonui5", "element");
+  private errorHandler = new ErrorHandler();
```

Comment:
class instances should be PascalCase
```suggestion
  private ErrorHandler = new ErrorHandler();
```

Reply:
done